pilot/lib/pilot/tester_manager.rb

```diff
@@ -43,6 +40,27 @@ def add_tester(options)
       end
     end
 
+    def add_tester_to_groups(tester: nil, app: nil)
+      default_external_group = app.default_external_group
+      if default_external_group.nil? && config[:groups].nil?
+        UI.user_error!("The app #{app.name} does not have a default external group.")
+        UI.user_error!("Please make sure to pass group names to the `:groups` option.")
```

Comment:
`user_error!` raises the exception, this code line won't be reached. Try the following instead:
```ruby
UI.user_error!("The app #{app.name} does not have a default external group. Please make sure to pass group names to the `:groups` option.")
```